backend/src/modules/tournament-tree/core/domain/TournamentTree.ts

```diff
@@ -26,19 +28,25 @@ export class TournamentTree {
       tournamentTeams: props.tournamentTeams,
       entityIdGenerator: props.entityIdGenerator,
     });
+
+    const tournamentTreeArray = winnerTree.getTournamentTreeArray().map((item, index) => {
+      return { ...item, matchNumberInSequence: index + 1 };
+    });
+
     const tournamentTreeProps = {
       tournamentId: props.tournamentId,
-      tournamentTreeArray: winnerTree.getTournamentTreeArray(),
+      tournamentTreeArray: tournamentTreeArray,
       tournamentTeams: props.tournamentTeams,
     };
+
     return new TournamentTree(tournamentTreeProps);
   }
 
-  static setTournamentTreeFromDataBase(props: {
+  public static setTournamentTreeFromDataBase(props: {
```

Comment:
podobnie jak wyżej :)

Reply:
Korzysta funkcja zewnętrzna z bazy danych.
Dodałem tylko to dla formalności. Bez słowa public też działa, a może sugerować, ze nie wiedomo czy jest publiczna czy prywatna